tests/test_test_utils.py

```diff
@@ -44,3 +50,126 @@ def test_should_not_fail_if_it_gets_a_collection_or_object(self):
         # It should not fail but the name does not have to be sensibly formatted.
         # We don't want to rely here on its exact structure.
         self.assertTrue(name.startswith("dummy_function_777_"))
+
+
+class TestTestUtilsBuildTestVideo(TestCase):
```

Comment:
This test class does contain a test that generates a video and checks its metadata to see if it was generated correctly (`test_generate_sample_video`).

It **also** contains tests that check the command because there's a separate function doing it and they're unit tests for that function. When I was writing it I first wrote down the command I was expecting then wrote a unit test based on it and then implemented a function that generates the command. It's just TDD.

I agree that one of these tests is an integration test and the other is a unit test and mixing them in a single class is not that great. Maybe I should just put the test that generates a video in a separate class called `TestTestUtilsBuildTestVideoIntegration`? I did not want to put it in `test_integration` because it's a utility that's used by integration tests and should be already tested by the time we run them :)

Reply:
I think we can leave it as is